build_tools/travis/install.sh

```diff
@@ -20,9 +20,9 @@ conda update --yes conda
 
 conda init bash
 conda create -n testenv -yq
-conda install -n testenv -yq python=3.8 numpy scipy scikit-learn matplotlib pandas pytest pytest-xdist joblib nibabel cython requests
```

Comment:
I think we don't use travis anymore? there may be some cleanup that we can do in another PR